### PR TITLE
Adding possibility to produce  through publishing events with fs2.Topic

### PR DESCRIPTION
**Pull request by lightsaway**
Hi, looked into the lib. 
I think that can be useful feature. I would like to mount producing stream in Http4s application as a stream, and produce from resources or other streams by publishing events from a topic

### CODE REVIEW

File: src/main/scala/com/ovoenergy/fs2/kafka/Producing.scala [hunk 1]
```diff
@@ -138,6 +145,19 @@ object Producing {
     }
   }
 
+  private[kafka] final class SubscribedProducePurtiallyApplied[F[_]] {
```
Comment (filosganga) [inline src/main/scala/com/ovoenergy/fs2/kafka/Producing.scala]: typo: Partially

Comment (lightsaway) [reply src/main/scala/com/ovoenergy/fs2/kafka/Producing.scala]: oh dammit) Will fix it

**Comment by lightsaway** [comment]
Fixed typos and "force" pushed

**Comment by filosganga** [comment]
@lightsaway Thanks!

**Pull request merged**